contracts/services/QueryService.sol

```diff
@@ -517,12 +517,13 @@ contract QueryService is IQueryService {
         uint256 tokenAmount,
         IAssetCommon token,
         IToken stablecoin,
-        uint256 price
+        uint256 price,
+        uint256 priceDecimals
```

Comment:
Why use `uint256`?!

Reply:
It's a read-only service so any optimization is pointless but I'll fix it in the new commit with other changes